resources/js/Directives/getinvolvedDirective.js

```diff
@@ -31,6 +31,13 @@ app.directive('getinvolved', ['$http', function ($http) {
             $scope.loadMore = function () {
                 $scope.totalDisplayed += 20;
             };
+
+            $scope.nomoreItems = function () {
+                if($scope.totalDisplayed > 288 ) {
```

Comment:
What is `288` ?

Reply:
@yashovardhanagrawal you could check the length of `self.contributors` instead